### PR TITLE
Add install enterprise --version flag to override installed enterprise version

### PR DESCRIPTION
**Pull request by ashleywang1**
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/1863

### CODE REVIEW

**Comment by marcogschmidt** [comment]
We should add this also to the open source install command, it's as easy as adding another `if` clause [here](https://github.com/solo-io/gloo/blob/6d972a451fc15c051bf5eba51c910ea049971c1f/projects/gloo/cli/pkg/cmd/install/installer.go#L190) and interpolating using the provided flag.

**Comment by ashleywang1** [comment]
I'm waiting for the semver constraints to be relaxed so that this doesn't force us to release gloo 1.3.0

File: projects/gloo/cli/pkg/flagutils/install.go [hunk 1]
```diff
@@ -18,7 +18,9 @@ func AddInstallFlags(set *pflag.FlagSet, install *options.Install) {
 }
 
 func AddEnterpriseInstallFlags(set *pflag.FlagSet, install *options.Install) {
-	set.StringVar(&install.LicenseKey, "license-key", "", "License key to activate GlooE features")
+	set.StringVar(&install.Enterprise.LicenseKey, "license-key", "", "License key to activate GlooE features")
+	set.StringVar(&install.Enterprise.Version, "version", "",
```
Comment (GrahamGoudeau) [inline projects/gloo/cli/pkg/flagutils/install.go]: Should this be `"release"`, for consistency with other existing flags? eg `release-name` in the install tree, and `release` in the upgrade tree

File: projects/gloo/cli/pkg/flagutils/install.go [hunk 2]
```diff
@@ -18,7 +18,9 @@ func AddInstallFlags(set *pflag.FlagSet, install *options.Install) {
...
+		"Version of gloo-ee to install (overrides the latest enterprise release)")
```
Comment (GrahamGoudeau) [inline projects/gloo/cli/pkg/flagutils/install.go]:
```suggestion
		"Version of Gloo Enterprise to install (defaults to latest)")
```

**Pull request closed without merging**